Makefile

```diff
@@ -239,6 +239,38 @@ check-gomin:
 		echo "Please update the Go version in 'doc/requirements.md' to be $(GOMIN) instead of $(DOC_GOMIN)"; \
 		exit 1; \
 	fi
+	@echo "Check copilot-instructions.md mentions the right Go minimum version"
+	$(eval COPILOT_GOMIN := $(shell sed -n 's/^LXD uses Go version \([0-9.]\+\).*/\1/p' .github/copilot-instructions.md))
+	if [ "$(COPILOT_GOMIN)" != "$(GOMIN)" ]; then \
+		echo "Please update the Go version in '.github/copilot-instructions.md' to be $(GOMIN) instead of $(COPILOT_GOMIN)"; \
+		exit 1; \
+	fi
+
+.PHONY: update-gomin
+update-gomin:
+ifndef NEW_GOMIN
+	@echo "Usage: make update-gomin NEW_GOMIN=1.x.y"
+	@echo "Current Go minimum version: $(GOMIN)"
+	exit 1
+endif
+	@echo "Updating Go minimum version from $(GOMIN) to $(NEW_GOMIN)"
+	
+	@# Update GOMIN in Makefile
+	sed -i 's/^GOMIN=.*/GOMIN=$(NEW_GOMIN)/' Makefile
+	
+	@# Update doc/requirements.md
+	sed -i 's/^LXD requires Go [0-9.]* or higher/LXD requires Go $(NEW_GOMIN) or higher/' doc/requirements.md
+	
+	@# Update .github/copilot-instructions.md
+	sed -i 's/^LXD uses Go version [0-9.]*/LXD uses Go version $(NEW_GOMIN)/' .github/copilot-instructions.md
+	
+	@echo "Go minimum version updated to $(NEW_GOMIN)"
+	if [ -t 0 ] && ! git diff --quiet -- ./Makefile ./doc/requirements.md ./.github/copilot-instructions.md; then \
```

Comment:
As mentioned before, for this target to be successful, at this point, all 3 files need to have been altered so I require for all 3 to have a diff.